test/common/formatter/substitution_formatter_test.cc

```diff
@@ -1613,6 +1613,38 @@ TEST(SubstitutionFormatterTest, StructFormatterNestedObject) {
   EXPECT_TRUE(TestUtility::protoEqual(out_struct, expected));
 }
 
+TEST(SubstitutionFormatterTest, StructFormatterList) {
```

Comment:
Can you also add test for list (or struct) nested in a list, please? Similar to the struct formatter tests.

Reply:
Done.